packages/core/src/sdk/analytics/platform/vtex/search.ts

```diff
@@ -14,27 +15,27 @@ const randomUUID = () =>
     ? crypto.randomUUID()
     : (Math.random() * 1e6).toFixed(0)
 
-const createStorage = (key: string, expiresSecond: number) => {
-  const timelapsed = (past: number) => (Date.now() - past) / 1e3
+const createCookie = (key: string, expiresSecond: number) => {
+  const urlDomain = `.${new URL(config.storeUrl).hostname}`
```

Comment:
Can you add a comment explaining this line? Why the . at the beggining? Why hostname?

Reply:
@icazevedo thanks for reviewing it 🎉 

I added some explanations [here](https://github.com/vtex/faststore/pull/2012/commits/85741c5a3a1cf0bb8db1a6c7a9e80d23133b1e62). Can you check pls :)